### PR TITLE
WIP: Add example contribution

### PR DESCRIPTION
**Pull request by LiyouZhou**
Contributed yolo_v2_tiny as an example. The contribution includes an
example inference pipeline using TVM.

depends on https://github.com/autowarefoundation/modelzoo/pull/2

### CODE REVIEW

**Comment by LiyouZhou** [comment]
@sgermanserrano need some help pushing some binaries to this PR.

**Comment by taichiH** [comment]
@LiyouZhou 
Hi, I'm from TierIV perception team.
We are currently developing tvm based lidar obstacle detection and here is a sample ros node. 
https://github.com/tier4/lidar_instance_segmentation_tvm

Currently, tvm inference files are generated by our own script from pytorch model, but I want to much it with the modelzoo inference pipeline format.

I think we can help you.

**Comment by LiyouZhou** [comment]
@taichiH Would love to have more contributions in this space. Would you like to have a chat and see where we can collaborate? If you don't mind, can you send an email to liyou.zhou@arm.com so we can set something up?

**Comment by LiyouZhou** [comment]
@JWhitleyWork do you have any experience with applying license for the pre-trained weights? The data license is clear, but what should we use for the trained model files?

**Comment by JWhitleyWork** [comment]
@LiyouZhou Our default is Apache 2.0 and I think that's probably fine for the weights files too. It's almost as permissive as MIT but has more legal-eze.

File: perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+```bash
+$ export MODEL_DIR=`pwd`/perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/
+$ mkdir ${MODEL_DIR}/example_pipeline/build
+$ docker run \
+    -it --rm \
+    -v ${MODEL_DIR}:${MODEL_DIR} -w ${MODEL_DIR} \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    autoware-model-zoo/tvm_cli:local \
+        --config ${MODEL_DIR}/definition.yaml \
+        --output_path ${MODEL_DIR}/example_pipeline/build
```
Comment (JWhitleyWork) [inline perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/README.md:32]: I would prefer if we put these Docker containers in https://hub.docker.com/u/autoware so people know they are Autoware-provided. I'm fine giving you push access to the repo. Are you OK with this?

Comment (LiyouZhou) [reply perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/README.md:32]: @JWhitleyWork I think that is the best way forward. Thank you.

Comment (LiyouZhou) [reply perception/camera_obstacle_detection/yolo_v2_tiny/tensorflow_fp32_coco/README.md:32]: Once they are in, I can push another patch that changes these docs.

**Review by JWhitleyWork — APPROVED** [review]

**Comment by JWhitleyWork** [comment]
Oops. I just realized this was still listed as WIP. Sorry about that.

On another note: https://hub.docker.com/r/autoware/model-zoo-tvm-cli/tags

**Comment by LiyouZhou** [comment]
@JWhitleyWork No worries, we will add more stuff in a following PR. Thank you for reviewing.

**Comment by LiyouZhou** [comment]
> Oops. I just realized this was still listed as WIP. Sorry about that.
> 
> On another note: https://hub.docker.com/r/autoware/model-zoo-tvm-cli/tags

Fantastic!

**Pull request merged**